### PR TITLE
Update README with Icecast/live mp3 support.

### PR DESCRIPTION
**Pull request by ryanginn**
I was really stuck with this myself, so I want to make sure newcomers can understand how to do it from now 😉

(also, please **remember** to do ``sudo apt-get install sox`` and ``sudo apt-get install libsox-fmt-mp3``)

### CODE REVIEW

**Comment by ryanginn** [comment]
@miegl

File: README.md [hunk 1]
```diff
@@ -102,12 +102,16 @@ If you use the argument `--audio -`, PiFmAdv reads audio data on standard input.
 sox -t mp3 http://www.linuxvoice.com/episodes/lv_s02e01.mp3 -t wav -  | sudo ./pi_fm_adv --audio -
```
Comment (miegl) [inline README.md:102]: This exact use case is described, here, why should it be included twice?

Comment (ryanginn) [reply README.md:102]: Yes, but it only specifies **pre recorded** mp3 files, not mentioning support for live mp3s.

Comment (miegl) [reply README.md:102]: This has nothing to do with the project though, you can change the wording so it's clear sox supports reading mp3 streams, however there should be no duplicates.

Comment (ryanginn) [reply README.md:102]: Alright, I will as soon as possible.

**Pull request closed without merging**